src/index.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable import/no-cycle, import/no-mutable-exports */
+import './style.css';
+import more from './more.png';
+import trash from './trash.png';
+import reload from './reload.png';
+import enter from './enter.png';
+import { check } from './status.js';
+import {
+  addCrud,
+  clearCrud,
+  editCrud,
+  removeCrud,
+} from './crud.js';
+
+// Object Template
+let items = [];
+class AddItem {
+  constructor(description, index) {
+    this.description = description;
+    this.completed = false;
+    this.index = index;
+    this.id = `item-${this.index}`;
+  }
+}
+
+// Storage
+const setStored = () => localStorage.setItem('newItems', JSON.stringify(items));
+const getStored = () => JSON.parse(window.localStorage.getItem('newItems'));
+
+// Display Task
+const checkList = document.querySelector('.checklist');
+function displayList() {
+  const list = getStored();
+  if (list !== null && list !== undefined) {
+    items = list;
+  }
+  const oldList = document.querySelectorAll('.container');
+  [...oldList].forEach((e) => e.remove());
+  items.forEach((item, i) => {
+    items[i].index = i;
+    const container = document.createElement('div');
+    const input = document.createElement('input');
+    const label = document.createElement('label');
+    const moreIcon = document.createElement('img');
+    moreIcon.src = more;
+    const trashIcon = document.createElement('img');
+    trashIcon.src = trash;
+    container.appendChild(input);
+    container.appendChild(label);
+    container.appendChild(moreIcon);
+    container.appendChild(trashIcon);
+    container.classList.add('container');
+    input.classList.add('checkbox');
+    label.classList.add('edit');
+    trashIcon.id = item.id;
+    input.setAttribute('type', 'checkbox');
+    label.setAttribute('contenteditable', 'true');
+    const textNode = document.createTextNode(item.description);
+    label.appendChild(textNode);
+    checkList.appendChild(container);
```

Comment:
This is all very meticulously written and excellently labelled with great variable names 👍  but do you think it might be quicker to write and a little easier to read if you use string literals with embedded variables?  It may also require less manipulation of the DOM.